pkg/api/core/service_serviceaccount.go

```diff
@@ -152,6 +152,55 @@ func (c *serviceAccount) CreateAlbIngressControllerServiceAccount(_ context.Cont
 	return account, nil
 }
 
+// nolint: lll
+func (c *serviceAccount) CreateAWSLoadBalancerControllerServiceAccount(_ context.Context, opts api.CreateAWSLoadBalancerControllerServiceAccountOpts) (*api.ServiceAccount, error) {
+	err := opts.Validate()
+	if err != nil {
+		return nil, errInvalidInputs(err)
+	}
+
+	config, err := clusterconfig.NewAWSLoadBalancerControllerServiceAccount(
+		opts.ID.ClusterName,
+		opts.ID.Region,
+		opts.PolicyArn,
+		v1alpha1.PermissionsBoundaryARN(opts.ID.AWSAccountID),
+	)
+	if err != nil {
+		return nil, errBuildServiceAccount(err)
+	}
+
+	account, err := c.createServiceAccount(opts.CreateServiceAccountOpts, config)
+	if err != nil {
+		return nil, errCreateServiceAccount(err)
+	}
+
+	return account, nil
+}
+
+func (c *serviceAccount) DeleteAWSLoadBalancerControllerServiceAccount(_ context.Context, id api.ID) error {
+	err := id.Validate()
+	if err != nil {
+		return errInvalidInputs(err)
+	}
+
+	config, err := clusterconfig.NewAWSLoadBalancerControllerServiceAccount(
```

Comment:
Why do we create a service account just to delete it (line 197)? Or does this function just create some kind of config despite its name

Reply:
I am not happy about this functionality, but essentially we are using `eksctl` to delete the service account for us. To do this, we need to provide a config the cluster name, region, service account name, etc.

If you look at the package name: `clusterconfig` you can see that your assumption is correct.

Reply:
Ah. I thought clusterconfig was our code. My bad